src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -155,6 +160,10 @@ def path(self) -> Path:
             )
         return flow_file
```

Comment:
Will this 'path' property raise exception when eager flow defined without flow dag yaml?

Reply:
Offline synced.